src/uu/install/src/install.rs

```diff
@@ -468,7 +471,19 @@ fn standard(mut paths: Vec<String>, b: &Behavior) -> UResult<()> {
         }
 
         if target.is_file() || is_new_file_path(&target) {
-            copy(&sources[0], &target, b)
+            copy(
+                sources.get(0).ok_or_else(|| {
+                    UUsageError::new(
```

Comment:
No coverage here, could you please add a test to trigger this error ? 
thanks

Reply:
Sorry about that! Let me know if there's anything else I should do as well :)